### PR TITLE
Type declarations for use-prefers-color-scheme

### PR DESCRIPTION
**Pull request by martonlederer**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

File: types/use-prefers-color-scheme/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Type definitions for use-color-scheme 0.1
+// Project: https://github.com/rfoel/use-prefers-color-scheme
+// Definitions by: Marton Lederer <https://github.com/martonlederer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function usePrefersColorScheme(): "no-preference" | "light" | "dark";
+export = usePrefersColorScheme;
```
Comment (amcasey) [inline types/use-prefers-color-scheme/index.d.ts:7]: Based on the underlying library, it seems like you want `export default usePrefersColorScheme`.

File: types/use-prefers-color-scheme/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      "strictFunctionTypes": true,
+      "strictNullChecks": true,
+      "baseUrl": "../",
+      "typeRoots": [
+          "../"
+      ],
+      "types": [],
+      "noEmit": true,
+      "forceConsistentCasingInFileNames": true,
+      "esModuleInterop": true
```
Comment (amcasey) [inline types/use-prefers-color-scheme/tsconfig.json:19]: If you `export default`, this shouldn't be necessary.

**Review by amcasey — CHANGES_REQUESTED** [review]

**Comment by martonlederer** [comment]
@amcasey the source module uses `module.exports = `, you can see [here](https://dev.azure.com/definitelytyped/DefinitelyTyped/_build/results?buildId=85810&view=logs&j=8635df7d-ecde-52a7-c7a8-84b998e35690&t=c264d665-0b79-59b7-7e04-9965cbe74860), that it fails if I try to use your suggestions.

**Comment by amcasey** [comment]
@martonlederer My mistake - sorry about that.  I looked [here](https://unpkg.com/browse/use-prefers-color-scheme@0.1.1/src/index.js) when I should have looked [here](https://unpkg.com/browse/use-prefers-color-scheme@0.1.1/dist/index.js).

**Comment by amcasey** [comment]
Hmm, that still doesn't explain why `esModuleInterop` was required though.  Can it be removed?

**Comment by martonlederer** [comment]
Just checked @amcasey, works perfectly without it, removed

**Review by amcasey — APPROVED** [review]
Thanks!  Sorry again for the confusion.

**Comment by martonlederer** [comment]
Ready to merge

**Pull request merged**